wrappers/_shared.nix

```diff
@@ -29,20 +29,23 @@ let
   extraFiles = lib.filter (file: file.enable) (lib.attrValues cfg.extraFiles);
 in
 {
-  options = {
-    nixvim.helpers = mkOption {
-      type = mkOptionType {
-        name = "helpers";
-        description = "Helpers that can be used when writing nixvim configs";
-        check = isAttrs;
-      };
-      description = "Use this option to access the helpers";
-    };
-  };
```

Comment:
We have no definition of `nixvim.lib` now, it is going to be merged in the `hm` or `nixos` lib? What about standalone, without the option?

Reply:
It's very confusing having so many things named `lib`!